RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -38,3 +38,5 @@
                             ConvStep*
                             conversionStepTracks
                             )
+_iterTracking_Phase2PU140 = iterTracking.copyAndExclude([InitialStepPreSplitting])
+eras.trackingPhase2PU140.toReplaceWith(iterTracking, _iterTracking_Phase2PU140)
```

Comment:
You need to keep `InitialStepPreSplitting` also for phase2, in the same way and for the same reason as it is used in `trackingPhase1PU70`. See https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py#L176